src/vectorsynchronizer.js

```diff
@@ -96,14 +96,15 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
   var onRemoveFeature = goog.bind(function(feature) {
     var geometry = feature.getGeometry();
     var id = goog.getUid(feature);
-    if (goog.isDefAndNotNull(geometry) && geometry.getType() == 'Point') {
+    if (!geometry || geometry.getType() == 'Point') {
       var context = csPrimitives.context;
-      var bbs = context.billboards;
       var bb = context.featureToCesiumMap[id];
       delete context.featureToCesiumMap[id];
       if (goog.isDefAndNotNull(bb)) {
-        goog.asserts.assertInstanceof(bb, Cesium.Billboard);
-        bbs.remove(bb);
+        var bbs = context.billboards;
+        if (bb instanceof Cesium.Billboard) {
+          bbs.remove(bb);
+        }
```

Comment:
I think you can simplify the code:

Instead of

```js
  if (goog.isDefAndNotNull(bb)) {
    var bbs = context.billboards;
    if (bb instanceof Cesium.Billboard) {
      bbs.remove(bb);
    }
  }
```

you can use:

```js
  if (bb instanceof Cesium.Billboard) {
    context.billboards.remove(bb);
  }
```